internal/camera/camera_test.go

```diff
@@ -59,9 +59,9 @@ func TestCameraConfigFromYaml(t *testing.T) {
 }
 
 func TestCameraNew(t *testing.T) {
-	var tests = []struct {
-		Description string
+	tests := []struct {
 		Expected    *Camera
+		Description string
```

Comment:
While not needed for tests, I always make sure structs are fieldaligned (there's a check for it and vs-code has a nice helper to do it for you):

- https://go101.org/article/memory-layout.html
- https://itnext.io/structure-size-optimization-in-golang-alignment-padding-more-effective-memory-layout-linters-fffdcba27c61
- https://dave.cheney.net/2015/10/09/padding-is-hard